definitions.yaml

```diff
@@ -237,6 +237,154 @@ definitions:
         type: object
         additionalProperties: true
 
+  V4GetClusterStatusResponse:
+    type: object
+    description: Object about a cluster's current state
+    properties:
+      aws:
+        type: object
+        description: |
+          AWS specific status information. Only given in AWS installations.
+        properties:
+          availabilityZones:
+            type: array
+            items:
+              type: object
+              properties:
+                name:
+                  type: string
+                  description: Name of the randomly selected availability zone
+                subnet:
+                  type: object
+                  properties:
+                    private:
+                      type: object
+                      properties:
+                        cidr:
+                          type: string
+                          description: |
+                            Private subnet CIDR allocated for this availability
+                            zone
+                    public:
+                      type: object
+                      properties:
+                        cidr:
+                          type: string
+                          description: |
+                            Public subnet CIDR allocated for this availability
+                            zone
+      cluster:
+        type: object
+        properties:
+          conditions:
+            type: array
+            items:
+              type: object
+              description: |
+                List of cluster conditions describing a cluster's state
+                transitions.
+              properties:
+                lastHeartbeatTime:
+                  type: string
+                  description: Last time the condition was checked and verified
+                lastTransitionTime:
+                  type: string
+                  description: |
+                    Last time the condition transitioned from one state to
+                    another
+                status:
+                  type: string
+                  description: |
+                    Status of the condition. One of True, False, or Unknown.
+                type:
+                  type: string
+                  description: |
+                    Condition type describing what kind of condition this is
+                    about. One of Creating, Created, Updating, Updated, or
+                    Deleting.
+          network:
+            type: object
+            properties:
+              cidr:
+                type: string
+                description: |
+                  The whole network CIDR for the cluster. Divided into multiple
```

Comment:
FYI: In our API docs it's fairly common to only talk about a "cluster" when we in fact mean the tenant cluster, because there is hardly any reason to talk about the control plane in this context.

Reply:
Oh, then this suggestion is not correct. Ignore it :flushed: 

Thanks for notifying about this!